bitnami/grafana-loki/templates/_helpers.tpl

```diff
@@ -0,0 +1,353 @@
+{{/*
+Return the proper Grafana Loki image name
+*/}}
+{{- define "grafana-loki.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.loki.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper Grafana Loki compactor fullname
+*/}}
+{{- define "grafana-loki.compactor.fullname" -}}
+{{- printf "%s-%s" (include "common.names.fullname" .) "compactor" -}}
```

Comment:
Should all these `grafana-loki.XXXX.fullname` be limited to 53 ?

Reply:
Change to trunc to 63 chars as we have in the common library